ros_ign_bridge/CMakeLists.txt

```
@@ -106,45 +106,81 @@ endforeach()
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
-endif()
 
-# TODO(anyone) Uncomment parts as they're ported to ROS 2
-#
-# # Tests
-# find_package(rostest REQUIRED)
-#
-# set(test_publishers
-#   ign_publisher
-#   ros_publisher
-# )
-#
-# set(test_subscribers
-#   ign_subscriber
-#   ros_subscriber
-# )
-#
-# foreach(test_publisher ${test_publishers})
-#   add_executable(${test_publisher}
-#     test/publishers/${test_publisher}.cpp
-#   )
-#   target_link_libraries(${test_publisher}
-#     ${catkin_LIBRARIES}
-#     ignition-msgs${IGN_MSGS_VER}::core
-#     ignition-transport${IGN_TRANSPORT_VER}::core
-#     gtest
-#     gtest_main
-#   )
-# endforeach(test_publisher)
-#
-# foreach(test_subscriber ${test_subscribers})
-#   add_rostest_gtest(test_${test_subscriber}
-#     test/${test_subscriber}.test
-#     test/subscribers/${test_subscriber}.cpp)
-#   target_link_libraries(test_${test_subscriber}
-#     ${catkin_LIBRARIES}
-#     ignition-msgs${IGN_MSGS_VER}::core
-#     ignition-transport${IGN_TRANSPORT_VER}::core
-#   )
-# endforeach(test_subscriber)
+  find_package(ament_cmake_gtest REQUIRED)
+  find_package(launch_testing_ament_cmake REQUIRED)
+  ament_find_gtest()
+
+  add_executable(test_ros2_publisher test/publishers/ros_publisher.cpp)
```

Can you remove all the `2` from `ros2`? A while back we updated all the API so that it was [ROS-version agnostic](https://community.gazebosim.org/t/renaming-ros-ign-packages/420). This makes it easier to port code across ROS 1 and 2.

```suggestion
  add_executable(test_ros_publisher test/publishers/ros_publisher.cpp)
```